packages/eslint-plugin-template/tests/rules/no-duplicate-attributes.test.ts

```diff
@@ -0,0 +1,163 @@
+        <input (change)="foo($event)" (change)="bar($event)">
+               ~~~~~~~~~~~~~~~~~~~~~~ ^^^^^^^^^^^^^^^^^^^^^^
+      `,
+      messages: [
+        { char: '~', messageId },
+        { char: '^', messageId },
+      ],
+    }),
+    convertAnnotatedSourceToFailureCase({
+      description: 'should fail with 2 banana in a box with the same name',
+      annotatedSource: `
+        <input [(ngModel)]="model" [(ngModel)]="otherModel">
+               ~~~~~~~~~~~~~~~~~~~ ^^^^^^^^^^^^^^^^^^^^^^^^
+      `,
+      messages: [
+        { char: '~', messageId },
+        { char: '^', messageId },
+      ],
+    }),
+    convertAnnotatedSourceToFailureCase({
+      description:
+        'should fail with duplicate attributes but allow non duplicates',
+      annotatedSource: `
+        <input [name]="foo" [other]="bam" [name]="bar">
+               ~~~~~~~~~~~~               ^^^^^^^^^^^^
+      `,
+      messages: [
+        { char: '~', messageId },
+        { char: '^', messageId },
+      ],
+    }),
+    convertAnnotatedSourceToFailureCase({
+      description: 'should fail with 3 duplications',
+      annotatedSource: `
+        <input [name]="foo" [name]="bar" [name]="bam">
+               ~~~~~~~~~~~~ ^^^^^^^^^^^^ ############
+      `,
+      messages: [
+        { char: '~', messageId },
+        { char: '^', messageId },
+        { char: '#', messageId },
+      ],
+    }),
+    convertAnnotatedSourceToFailureCase({
+      description: 'should fail multiple combinations of duplicates',
+      annotatedSource: `
+        <input [(ngModel)]="model" [name]="foo" [(ngModel)]="otherModel" name="bar">
+               ~~~~~~~~~~~~~~~~~~~ ^^^^^^^^^^^^ ######################## %%%%%%%%%%
+      `,
+      messages: [
+        { char: '~', messageId },
+        { char: '^', messageId },
+        { char: '#', messageId },
+        { char: '%', messageId },
+      ],
+    }),
+    convertAnnotatedSourceToFailureCase({
+      description: 'should fail with multiple animation outputs',
+      annotatedSource: `
+        <input (@fade.start)="animationStarted($event)" (@fade.start)="animationStarted($event)">
+               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+      `,
+      messages: [
+        { char: '~', messageId },
+        { char: '^', messageId },
+      ],
+    }),
+    convertAnnotatedSourceToFailureCase({
+      description: 'should fail with multiple outputs on the window',
+      annotatedSource: `
+        <input (window:resize)="windowResized($event)" (resize)="resize()" (window:resize)="windowResized2($event)">
+               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+      `,
+      messages: [
+        { char: '~', messageId },
+        { char: '^', messageId },
+      ],
+    }),
+    convertAnnotatedSourceToFailureCase({
+      description: 'not allow two way binding and output with the change event',
```

Comment:
nit: what about changing the description to something like this to maintain the standard that was adopted in other cases in this file:

```suggestion
      description: 'should fail with two way binding and its corresponding output',
```